rct229/data_fns/table_3_2_test.py

```diff
@@ -10,43 +10,45 @@
     check_enumeration_to_osstd_match_field_value_map,
     find_osstd_table_entry,
 )
+from rct229.schema.config import ureg
 
+btuh_per_ft2 = ureg("british_thermal_unit / foot ** 2 / planck_constant")
```

Comment:
No `planck_constant`. See above comment.
Clever use of `btuh`. Maybe use it in table_3_2_fns.py for consistency?